content-src/experiments/notes.yaml

```diff
@@ -0,0 +1,79 @@
+id: 14
+title: 'Notes'
+slug: notes
+platforms: ['addon']
+thumbnail: /static/images/experiments/notes/experiments_experiment/thumbnail.png
+description: >
+  A simple notepad built into Firefox. Ever need to jot
+  something down while you browse the web? This is the experiment for you.
+introduction: >
+  Notes adds a simple, one-page notepad to your browser’s sidebar. We’ve added a few basic formatting options to start, but we want your feedback to tell us what features you’d like to see us add.
+image_twitter: /static/images/experiments/min-vid/social/notes-twitter.jpg
+image_facebook: /static/images/experiments/min-vid/social/notes-fb.jpg
+changelog_url: 'https://github.com/mozilla/notes/blob/master/CHANGELOG.md'
```

Comment:
Nit: We should just point this to https://github.com/mozilla/notes/releases or something until we have a CHANGELOG.md file.